### PR TITLE
doc: add link to docker tutorials

### PR DESCRIPTION
**Pull request by generalova-kate**
### Details:
 - Add link to openVINO Docker tutorials located on GitHub openvinotoolkit/docker_ci repo

### CODE REVIEW

File: docs/install_guides/installing-openvino-docker-linux.md [hunk 1]
```diff
@@ -2,33 +2,39 @@
...
+Prebuilt images are available on:
+
 - [Docker Hub](https://hub.docker.com/u/openvino)
 - [Red Hat* Quay.io](https://quay.io/organization/openvino)
 - [Red Hat* Ecosystem Catalog](https://catalog.redhat.com/software/containers/intel/openvino-runtime/606ff4d7ecb5241699188fb3)
 
+## Build a Docker* Image
+
+You can use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles) or generate a Dockerfile with your setting via [DockerHub CI Framework](https://github.com/openvinotoolkit/docker_ci). The Framework can generate a Dockerfile, build, test, and deploy an image with the Intel® Distribution of OpenVINO™ toolkit.
+As well, you can try our [Tutorials](https://github.com/openvinotoolkit/docker_ci/tree/master/docs/tutorials) which demonstrate usage of OpenVINO™ Docker containers.
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-docker-linux.md:31]: As well, you can try our ---> You can also try our
demonstrate usage ---> demonstrate the usage

File: docs/install_guides/installing-openvino-docker-linux.md [hunk 2]
```diff
@@ -2,33 +2,39 @@
 
 The Intel® Distribution of OpenVINO™ toolkit quickly deploys applications and solutions that emulate human vision. Based on Convolutional Neural Networks (CNN), the toolkit extends computer vision (CV) workloads across Intel® hardware, maximizing performance. The Intel® Distribution of OpenVINO™ toolkit includes the Intel® Deep Learning Deployment Toolkit.  
 
-This guide provides the steps for creating a Docker* image with Intel® Distribution of OpenVINO™ toolkit for Linux* and further installation.  
+This guide provides device specific of creation a Docker* image with Intel® Distribution of OpenVINO™ toolkit for Linux* and further usage.
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-docker-linux.md:5]: device specific of creation a Docker* image ---> device specifics for a Docker\* image creation with Intel® Distribution of OpenVINO™ toolkit for Linux* and its further usage.

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 The Intel® Distribution of OpenVINO™ toolkit quickly deploys applications and solutions that emulate human vision. Based on Convolutional Neural Networks (CNN), the toolkit extends computer vision (CV) workloads across Intel® hardware, maximizing performance. The Intel® Distribution of OpenVINO™ toolkit includes the Intel® Deep Learning Deployment Toolkit.  
 
-This guide provides the steps for creating a Docker* image with Intel® Distribution of OpenVINO™ toolkit for Windows* and further installation.
+This guide provides device specific of creation a Docker* image with Intel® Distribution of OpenVINO™ toolkit for Windows* and further usage.
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-docker-windows.md:5]: device specific of creation ---> device specifics for a Docker* image creation with Intel® Distribution of OpenVINO™ toolkit for Linux* and its further usage.

File: docs/install_guides/installing-openvino-docker-linux.md (hunk 2 above)
Comment (generalova-kate) [inline docs/install_guides/installing-openvino-docker-linux.md:5]:
```suggestion
This guide provides device specifics for a Docker* image creation with Intel® Distribution of OpenVINO™ toolkit for Linux* and its further usage.  
```

File: docs/install_guides/installing-openvino-docker-linux.md [hunk 4]
```diff
@@ -2,33 +2,39 @@
...
+Prebuilt images are available on:
+
 - [Docker Hub](https://hub.docker.com/u/openvino)
 - [Red Hat* Quay.io](https://quay.io/organization/openvino)
 - [Red Hat* Ecosystem Catalog](https://catalog.redhat.com/software/containers/intel/openvino-runtime/606ff4d7ecb5241699188fb3)
 
+## Build a Docker* Image
+
+You can use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles) or generate a Dockerfile with your setting via [DockerHub CI Framework](https://github.com/openvinotoolkit/docker_ci). The Framework can generate a Dockerfile, build, test, and deploy an image with the Intel® Distribution of OpenVINO™ toolkit.
+As well, you can try our [Tutorials](https://github.com/openvinotoolkit/docker_ci/tree/master/docs/tutorials) which demonstrate usage of OpenVINO™ Docker containers.
```
Comment (generalova-kate) [inline docs/install_guides/installing-openvino-docker-linux.md:31]:
```suggestion
You can also try our [Tutorials](https://github.com/openvinotoolkit/docker_ci/tree/master/docs/tutorials) which demonstrate the usage of OpenVINO™ Docker containers.
```

File: docs/install_guides/installing-openvino-docker-windows.md (hunk 3 above)
Comment (generalova-kate) [inline docs/install_guides/installing-openvino-docker-windows.md:5]:
```suggestion
This guide provides device specifics for a Docker* image creation with Intel® Distribution of OpenVINO™ toolkit for Linux* and its further usage.
```

**Review by tsavina — APPROVED** [review]

**Pull request merged**